subgraphs/uniswap-forks/protocols/ubeswap/config/networks/celo/celo.ts

```diff
@@ -0,0 +1,92 @@
+import { Address, BigDecimal } from '@graphprotocol/graph-ts';
+import { Factory } from '../../../../../generated/Factory/Factory';
+import { FeeSwitch, Network, PROTOCOL_SCHEMA_VERSION, RewardIntervalType } from '../../../../../src/common/constants';
+import { Configurations } from '../../../../../configurations/configurations/interface';
+import { PROTOCOL_SUBGRAPH_VERSION, PROTOCOL_METHODOLOGY_VERSION, PROTOCOL_NAME, PROTOCOL_SLUG } from '../../../src/common/constants';
+import { toLowerCase, toLowerCaseList } from '../../../../../src/common/utils/utils';
+
+export class UbeswapCeloConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.CELO;
+  }
+  getProtocolName(): string {
+    return "Ubeswap";
+  }
+  getProtocolSlug(): string {
+    return "Ubeswap";
+  }
+
+  getSchemaVersion(): string {
+    return PROTOCOL_SCHEMA_VERSION;
+  }
+  getSubgraphVersion(): string {
+    return PROTOCOL_SUBGRAPH_VERSION;
+  }
+  getMethodologyVersion(): string {
+    return PROTOCOL_METHODOLOGY_VERSION;
+  }
+  getFactoryAddress(): string {
+    return "0x62d5b84be28a183abb507e125b384122d2c25fae";
+  }
+  getFactoryContract(): Factory { 
+    return Factory.bind(Address.fromString("0x62d5b84be28a183abb507e125b384122d2c25fae"));
+  }
+  getTradeFee(): BigDecimal {
+    return BigDecimal.fromString("3");
+  }
+  getProtocolFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("0.5");
+  }
+  getLPFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("2.5");
+  }
+  getProtocolFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0");
+  }
+  getLPFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("3");
+  }
+  getFeeOnOff(): string {
```

Comment:
Where did you see that this fee starts on? I have just seen that 0.3% goes to the liquidity providers and eventually some will be distributed to token holders.

Reply:
i see ,the trade fee is 0.3  and LP got 0.25%  the protocol got 0.05%